pkg/authz/oauth/constants.go

```diff
@@ -16,8 +16,14 @@ const (
 
 // IDP Auth type string const
 const (
-	IDPAuthTypeAccessToken = "accessToken"
-	IDPAuthTypeClient      = "client"
+	IDPAuthTypeAccessToken             = "accessToken"
+	IDPAuthTypeClient                  = "client"
+	IDPAuthTypeClientSecretBasic       = "client_secret_basic"
```

Comment:
this must be the default from the IDP but camelCase vs underscore_seperation bothers me a little